provider/resource_keycloak_attribute_importer_identity_provider_mapper_test.go

```diff
@@ -51,6 +51,32 @@ func TestAccKeycloakAttributeImporterIdentityProviderMapper_withExtraConfig(t *t
 	})
 }
 
+func TestAccKeycloakAttributeImporterIdentityProviderMapper_socialOidcProvider(t *testing.T) {
+	t.Parallel()
+	for _, providerId := range []string{"facebook", "github", "google", "instagram", "linkedin", "microsoft", "paypal", "stackoverflow"} {
+		providerId := providerId
+		t.Run("", func(t *testing.T) {
```

Comment:
nitpick: I'd add the `providerId` as the test name here

Reply:
Addressed in 7120bb4